src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1087,6 +1087,7 @@ namespace ReportSizingManager {
                                 AutosizeDes = max(ZoneEqSizing(CurZoneEqNum).AirVolFlow,
                                                   ZoneEqSizing(CurZoneEqNum).CoolingAirVolFlow,
                                                   ZoneEqSizing(CurZoneEqNum).HeatingAirVolFlow);
+                                // what is this IF block for? Already inside of (SizingType == CoolingAirFlowSizing).
```

Comment:
If this is true then the SizingType == HeatingAirflowSizing block should be removed from here.

Reply:
As long as others agree with my logic in seeing that it doesn't belong.